src/utransport.rs

```diff
@@ -514,33 +505,32 @@ impl UTransport for UPClientZenoh {
                 self.send_request(&zenoh_key, payload, attributes).await
             }
             UMessageType::UMESSAGE_TYPE_RESPONSE => {
-                Validators::Response
+                UAttributesValidators::Response
                     .validator()
                     .validate(&attributes)
                     .map_err(|e| {
-                        UStatus::fail_with_code(
-                            UCode::INVALID_ARGUMENT,
-                            format!("Wrong Response UAttributes {e:?}"),
-                        )
+                        let msg = format!("Wrong Response UAttributes: {e:?}");
+                        log::error!("{msg}");
+                        UStatus::fail_with_code(UCode::INVALID_ARGUMENT, msg)
                     })?;
                 // Get Zenoh key
                 let topic = attributes.clone().source;
```

Comment:
When I was reading the [`up-spec` changes](https://github.com/eclipse-uprotocol/up-spec/blob/c0ea3bfb994a0e4db46dafca0e7b9d6bd7ba2b3b/up-l1/zenoh.adoc#254-transfer-response-message) you had put together, in this section on Response messages, you mention you should use `sink`.

Shouldn't this be `sink` then?

Reply:
It seems like there is a little mismatch. Let me figure it out.

Reply:
Update the spec and code for alignment
https://github.com/eclipse-uprotocol/up-spec/blob/a3681e3a9f3bb824b75315f276281088c0c8e605/up-l1/zenoh.adoc#254-transfer-response-message